sklearn/multiclass.py

```diff
@@ -392,6 +394,87 @@ def intercept_(self):
         return np.array([e.intercept_.ravel() for e in self.estimators_])
 
 
+def _fit_regression(estimator, X, y):
+    estimator = clone(estimator)
+    estimator.fit(X, y)
+    return estimator
+
+
+def _parallel_helper(obj, methodname, *args, **kwargs):
+    """Private helper to workaround Python 2 pickle limitations"""
```

Comment:
My preference would be to move this to some shared location as this is a verbatim copy from `forest.py` instead of having this both here and there. Don't know where to put it though.